cime/scripts/lib/CIME/case/check_input_data.py

```diff
@@ -2,7 +2,11 @@
 API for checking input for testcase
 """
 from CIME.XML.standard_module_setup import *
+<<<<<<< HEAD
```

Comment:
This is definitely not right. Looks like conflict metadata was committed.

Reply:
Ok that should be fixed now.